### PR TITLE
Providers: Add WalletConnect as one of the available connectors

### PR DESCRIPTION
**Pull request by Evalir**
Adds WalletConnect and deprecates explicit Frame support.

_Note: WalletConnect only works on Mainnet. :(_

### CODE REVIEW

File: src/ethereum-providers/icons/WalletConnect.svg [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (sohkai) [inline src/ethereum-providers/icons/WalletConnect.svg:1]: Note that we could've probably optimized this asset :)

File: src/lib/web3-utils.js [hunk 2]
```diff
@@ -5,7 +5,7 @@ import { toChecksumAddress } from 'web3-utils'
 import env from '../environment'
 import { getDefaultChain } from '../local-settings'
 
-const DEFAULT_LOCAL_CHAIN = ''
+const DEFAULT_LOCAL_CHAIN = 'i'
```
Comment (sohkai) [inline src/lib/web3-utils.js:8]: Not sure, but should this be `1`?

File: src/ethereum-providers/icons/WalletConnect.svg (hunk 1 above)
Comment (Evalir) [reply src/ethereum-providers/icons/WalletConnect.svg:1]: In my list of to-dos today. :)

File: src/lib/web3-utils.js (hunk 2 above)
Comment (Evalir) [reply src/lib/web3-utils.js:8]: Actually, 1337 as it's a local chain. Fixed over in the `last-web-features` branch.

**Pull request merged**